src/main/java/de/uni_tuebingen/ub/nppm/model/SelektionTitelKritik.java

```diff
@@ -0,0 +1,31 @@
+package de.uni_tuebingen.ub.nppm.model;
+
+import javax.persistence.*;
+import java.util.*;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+@Entity
+@Table(name = "selektion_titelkritik")
+@Cacheable
+@org.hibernate.annotations.Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+public class SelektionTitelKritik extends SelektionBezeichnung{
+
+     @ManyToMany(mappedBy = "titelKritiken")
+    private Set<Einzelbeleg> einzelbeleg = new HashSet<>();
+
+    public Set<Einzelbeleg> getEinzelbeleg() {
```

Comment:
Since we get a List / Set, we should name the function getEinzelbelege(). The variable should be named as plural form as well.

Reply:
done